### ISSUE TITLE
Add cancel button to creation form of page, poi, event, organisation, language node, push notification and users

### ISSUE THREAD

**Issue by osmers** [digitalfabrik/integreat-cms#2930]
### Motivation
A clear button that lets users cancel the current creation process they started is more intuitive than simply clicking on a menu item to leave the form

### Proposed Solution
Add a red cancel button to each creation form listed above. Maybe also to the edit forms? At least for pages, pois and events we would have to discuss the effect of the button (i.e. what to do with automatic saves etc)

### Alternatives
Keep it as it is

### User Story
As a user of the CMS I want to have a clear button that let's me cancel the changes I have made in order to be sure that no changes are saved and nothing was created.

### Additional Context
Same button as for automatic translations bulk action: 
![image](https://github.com/user-attachments/assets/e8c3238b-d1ec-4e87-87f7-af0ee2e877ed)

**Comment by MizukiTemma** [comment]
Where should it redirect after clicking the button? Probably to the list (cancel on the organigation form -> the organization list)?

**Comment by osmers** [comment]
@MizukiTemma yes, exactly - basically back to the page where you clicked on the create button...

**Comment by lunars97** [comment]
@nikolahoff should we change the cancel button color to `ghost-button` (instead of red which is used for delete) and from now on use this color for all cancel buttons? :thinking:

**Comment by nikolahoff** [comment]
@lunars97 cancel button should have this behaviour like in this [design](https://www.figma.com/design/6U7R7Xj4wL7sbjxKRmOG9D/CMS-Project?node-id=2429-1129&t=KJi9rox7rgWeYceg-1) 
Delete buttons are different. For now we keep it like it is for delete. Maybe with the overall redesign we might change it.
